ui/tests/acceptance/enterprise-namespaces-test.js

```diff
@@ -58,14 +60,25 @@ module('Acceptance | Enterprise | namespaces', function (hooks) {
     await settled();
     await authPage.tokenInput('root').submit();
     await settled();
+
+    // Open the namespace picker & wait for it to render
     await click('[data-test-namespace-toggle]');
-    await waitFor('[aria-selected="true"]');
-    assert
-      .dom('[aria-selected="true"]')
-      .hasText('beep/boop', 'current namespace does not begin or end with /');
-    assert
-      .dom('[data-option-index="3"]')
-      .hasText('beep/boop/bop', 'shows the full namespace path in the toggle');
+    await waitFor('svg[data-test-icon="check"]');
```

Comment:
A pattern we're moving towards with data-test selectors, especially those that are commonly used, is to import them from `tests/helpers`. See general-selectors [here](https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/general-selectors.ts#L15) for the icon one and it being used [here](https://github.com/hashicorp/vault/blob/main/ui/tests/integration/components/tools/unwrap-test.js#L85).

A good rule of thumb is if you're hard-coding data-test-whatever into a test file, search test `tests/helpers` directory to see if it's already been defined.

Reply:
Oooo nice! Thank you for linking examples, this makes a lot of sense :) I'll update accordingly